site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,511 @@
+
+As promised, the latest release of `kctrl` introduces a set of commands which allow users to observe and interact with App CRs more conveniently. `kctrl` v0.36.1 also introduces a status tailing experience which surfaces relevant information while waiting for resources to reconcile.
+
+# The `app` commands
+These commands aim to help users manage App CRs on the cluster better. Lets dig in and see what they look like!
+
+The user can list apps created on the cluster by running,
+```bash
+$ kctrl app list
+Target cluster 'https://127.0.0.1:64709' (nodes: minikube)
+
+Available apps in namespace 'default'
+
+Name        Status               Since Deploy  Owner                     Age  
+hello-app   Reconcile succeeded  1m            PackageInstall/hello-app  1m  
+simple-app  Reconcile succeeded  22s           -                         1m  
+
+Succeeded
+```
+`kctrl` makes a distinction between App CRs deployed by users and the ones created due to a package installation. 
+
+We can take a closer look at an app by using the `get` command.
+```bash
+$ kctrl app get -a simple-app
+Target cluster 'https://127.0.0.1:64709' (nodes: minikube)
+
+Namespace         default  
+Name              hello-app  
+Service Account   hello-app-default-sa  
+Status            Reconcile succeeded  
+Owner References  - packaging.carvel.dev/v1alpha1/PackageInstall/hello-app  
+Conditions        - type: ReconcileSucceeded  
+                    status: "True"  
+                    reason: ""  
+                    message: ""  
+
+Succeeded
+```
+In case the app errors out, the command displays relevant errors. For instance, an app which is applying a resource with a misspelt "kind" field looks like:
+```bash
+$ kctrl app get -a faulty-app
+Target cluster 'https://127.0.0.1:64709' (nodes: minikube)
+
+Namespace             default  
+Name                  faulty-app  
+Service Account       default-ns-sa  
+Status                Reconcile failed: Deploying: Error (see .status.usefulErrorMessage for details)  
+Owner References      -  
+Conditions            - type: ReconcileFailed  
+                        status: "True"  
+                        reason: ""  
+                        message: 'Deploying: Error (see .status.usefulErrorMessage for details)'  
+Failing Stage         deploy  
+Useful Error Message  kapp: Error: Expected to find kind '/v1/ConfigMaps', but did not:  
+                      - Kubernetes API server did not have matching apiVersion + kind  
+                      - No matching CRD was found in given configuration  
+
+Succeeded
+```
+It highlights that the deploy stage failed, while displaying the error message.
+
+The `pause` and `kick` commands help users trigger and pause reconciliations for their apps.
```

Comment:
```suggestion
The `pause` and `kick` commands help users pause and trigger reconciliations for their apps.
```
Can we have this in the same order? 

Also instead of using `pause` second time, we can use `stop` or `halt` maybe. Okay, with this also just a thought.

Reply:
Makes sense, will do